suite-native/storage/src/storage.ts

```diff
@@ -1,23 +1,44 @@
 import { MMKV } from 'react-native-mmkv';
 
+import * as Random from 'expo-random';
+import * as SecureStore from 'expo-secure-store';
 import { Storage } from 'redux-persist';
 
-const storage = new MMKV({
-    id: 'trezorSuite-app-storage',
-    encryptionKey: 'FIXME',
-});
-
-export const mmkvStorage: Storage = {
-    setItem: (key, value) => {
-        storage.set(key, value);
-        return Promise.resolve(true);
-    },
-    getItem: key => {
-        const value = storage.getString(key);
-        return Promise.resolve(value);
-    },
-    removeItem: key => {
-        storage.delete(key);
-        return Promise.resolve();
-    },
+const ENCRYPTION_KEY = 'STORAGE_ENCRYPTION_KEY';
+
+export const purgeStorageEncryptionKey = () => SecureStore.deleteItemAsync(ENCRYPTION_KEY);
+
+const retrieveStorageEncryptionKey = async () => {
+    let secureKey = await SecureStore.getItemAsync(ENCRYPTION_KEY);
+
+    if (secureKey == null) {
+        secureKey = Buffer.from(Random.getRandomBytes(32)).toString('hex');
```

Comment:
just curious? Why 32? Is it enough?

Reply:
I used 32 bytes because I was convinced that MMKV is using AES256. Now I double-checked and according to this resource https://github.com/Tencent/MMKV/wiki/FAQ it is AES128 only. That means That 32 bytes are twice as much as needed since AES uses only 128 bits (16 bytes).

I believe that it is not a problem to have a longer key, so I will leave 32 bytes there. Agree?

Reply:
Honestly I have no idea. I would go down to 16 since having more probably don't have any extra benefits.

Reply:
changed to 16 bytes.